charts/gardener/gardenlet/values.yaml

```diff
@@ -16,6 +16,9 @@ global:
       limits:
         cpu: 2000m
         memory: 512Mi
+    metrics:
+      enabled: true
+      port: 2720
```

Comment:
Why is this now duplicated? Can you reuse `gardenlet.config.server.https.port`?
Alternatively, you might think about using a named port in the pod spec, if prometheus service discovery can also handle this.

Reply:
Prometheus can handle the named ports in the pod spec, but since we are using the annotations it does not matter. I've reused `gardenlet.config.server.https.port`